qubesappmenus/receive.py

```diff
@@ -159,7 +159,7 @@ def get_appmenus(vm):
     appmenus = {}
     line_rx = re.compile(
         r"([a-zA-Z0-9._-]+?)(?:\.desktop)?:"
-        r"([a-zA-Z0-9-]+(?:\[[a-zA-Z@_]+\])?)=(.*)")
+        r"([a-zA-Z0-9-]+(?:\[[a-zA-Z@_]+\])?)\s*=(.*)")
```

Comment:
if you add `\s*` on the other side of `=` too, I think you can avoid all those `strip()` below, as regex will do that for you.

Reply:
One strip is still necessary, for whitespace at the end of lines; other than that, I pushed a fix.